### ISSUE TITLE
Design and implement a debriefing screen

### ISSUE THREAD

**Issue by gajop** [ZeroK-RTS/Chobby#186]
A more detailed debriefing screen should exist.
It should show (ordered in importance):
- post game chat
- Winners/losers
- personal XP and elo gain (level gain too if it happened)
- game awards
- game stats/graphs (?)
- achievement progress (imo difficult/unnecessary)

**Comment by Licho1** [comment]
Achievements are part of the todo, along with avatar unlocks. Tbh I would wait with this until we finalize this other stuff, or we would have to redo it .

**Comment by GoogleFrog** [comment]
I like how postgame chat is currently on the MM screen. You are more likely to press the MM button again.

**Comment by gajop** [comment]
Yes. I noticed that as well and think that's important. My idea is to provide some sort of UI that would make that just as likely while not having to display all queues and take valuable space.
My suggestion would be to provide a "Queue again"/"Requeue" button that's enticing to click.
